### ISSUE TITLE
ID QA Review for PSYC 339

### ISSUE THREAD

**Issue by kmarjanovic** [TWUOnline/psyc339#9]
Course Units

U1
- [x] 1.1.1 Activity: What is Trauma? - 1st video isn't displaying'
- [x] [The Body Keeps the Score](https://twuonline.github.io/psyc339/assets/u1/BKTS_Prologue.pdf) - flagging accessibility & copyright issue. The pdfs are not accessible (& has fingers in the photocopies!).  I need to look through the course to see how much of the resource we are using.  So far it's just the Prologue & Chapter 3.
    1.1.2 Activity: Prologue  - I can look for excerpts e.g. [Prologue excerpt[(https://www.penguinrandomhouse.ca/books/313183/the-body-keeps-the-score-by-bessel-van-der-kolk-md/9780143127741/excerpt).  
For now, can we replace the The Body Keeps the Score links in 1.1.2 & 1.1.3 with the [The Body Keeps the Score - full text](https://ia601604.us.archive.org/35/items/the-body-keeps-the-score-pdf/The-Body-Keeps-the-Score-PDF.pdf)   
Production question - If we're able to use the chapters, can production split up this pdf so we only take the chapters we need?  We have this book in the library so I can also ask them to scan the chapters.
- [ ] 1.3.1 Activity - pdf not accessible [Treating Adult Survivors of Childhood Emotional Abuse and Neglect](https://twuonline.github.io/psyc339/assets/u1/Survivors_of_Emotional_Abuse_and_Neglect.pdf) (2021) - Kelly to find replacement
- [x] After checking your learning, there are Zotero references.  https://learn.twu.ca/mod/page/view.php?id=1312186.  Move to https://learn.twu.ca/mod/page/view.php?id=1312187 (replace those items as that is in the Overview page)

**Comment by kmarjanovic** [comment]
@chloecwh all remaining issues are quarto related so doesn't impact import.  Note many issues related to pdf accessibility so I'm enlisting the help of the library!  Remaining design issues on QA doc.

@emmanuelD-AFK @cmadland ID quality check complete.

**Comment by chloecwh** [comment]
Imported to [PSYC 339 OL - Mental Health and Adjustment (FA 2025)](https://learn.twu.ca/course/view.php?id=38976)

Gentle reminder to Emily
1. replace syllabus with latest information
2. update [Live Class Sessions](https://learn.twu.ca/course/view.php?id=38976&section=4) with Teams link and unhide topic
3. after review, change to "show" course visibility in course setting page

@emmanuelD-AFK

**Comment by chloecwh** [comment]
Reopening as I realize still a few pdfs to replace

**Comment by emmanuelD-AFK** [comment]
Do you advise starting communication with Emily?

**Comment by chloecwh** [comment]
- [x] 1.3.3 Emotion-Focused Family Therapy: A Transdiagnostic Model for Caregiver-Focused Interventions (2020) need to be replaced... I realized it was not with the right file, and the original doc or bookdown didn't gave any pdf content as well @kmarjanovic any suggestions on what to do? Zotero links to https://psycnet.apa.org/PsycBOOKS/toc/16145, which we don't have access to

**Comment by kmarjanovic** [comment]
- [x] @chloecwh we do have twu access to psycnet. Please link: 1.3.3 Emotion-Focused Family Therapy: A Transdiagnostic Model for Caregiver-Focused Interventions (2020) https://psycnet.apa.org/PsycBOOKS/toc/16145, which we don't have access to

**Comment by chloecwh** [comment]
Yes, added it! @kmarjanovic
